<?xml version="1.0"?>
<launch>

<arg name="load_gripper" default="true" />
  <include file="$(find franka_control)/launch/franka_control.launch" >
    <arg name="robot_ip" value="172.16.0.12" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <rosparam command="load" file="$(find franka_example_controllers)/config/franka_example_controllers.yaml" />
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"  args="kth_joint_pose_effort_interface_controller"/>


</launch>
